app/controllers/home_controller.rb

```diff
@@ -13,13 +13,10 @@ def index
   end
 
   def landing
-    recipe_images = RecipeImage.find([549, 376, 529, 526])
```

Comment:
How would a feature flag help here? I was thinking it should probably in a config some where and default to the last 4 recipes.